config/site_settings.yml

```diff
@@ -2325,14 +2324,6 @@ developer:
   instrument_gc_stat_per_request:
     default: false
     hidden: true
-  enable_experimental_admin_ui_groups:
```

Comment:
Since we are deleting this one (which is fine because we don't want it anymore anyway) we just need a migration to DELETE FROM site_settings WHERE name = 'enable_experimental_admin_ui_groups';